packages/react-components/react-datepicker-compat/src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -50,7 +50,9 @@ describe('DatePicker', () => {
   isConformant({
     Component: DatePicker,
     displayName: 'DatePicker',
-    disabledTests: ['consistent-callback-args'],
+    // component-has-root-ref is disabled because the root is an Input component, the conformance test thinks the
```

Comment:
Can you make an issue to fix the conformance test?

Reply:
Done! #28387